pkg/database/user.go

```diff
@@ -227,12 +227,16 @@ func (u *User) Stats(db *Database, realmID uint, start, stop time.Time) ([]*User
 
 // ListUsers returns a list of all users sorted by name.
 // Warning: This list may be large. Use Realm.ListUsers() to get users scoped to a realm.
-func (db *Database) ListUsers(p *pagination.PageParams, q string) ([]*User, *pagination.Paginator, error) {
+func (db *Database) ListUsers(p *pagination.PageParams, q string, systemAdminOnly bool) ([]*User, *pagination.Paginator, error) {
 	var users []*User
 	query := db.db.Model(&User{}).
 		Where("system_admin IS FALSE").
 		Order("LOWER(name) ASC")
 
+	if systemAdminOnly {
+		query = query.Where("system_admin IS TRUE")
```

Comment:
does this work?

will have system_admin IS FALSE followed by sytem_admin is true

better to make `systemAdminOnly` a param to the query?

Reply:
Sorry I think that was a merge conflict with: https://github.com/google/exposure-notifications-verification-server/pull/985